bundle/edu.gemini.auxfile.workflow/src/main/java/edu/gemini/auxfile/copier/osgi/OsgiCopyConfig.java

```diff
@@ -38,7 +41,9 @@ private OsgiCopyConfig(AuxFileType type, BundleContext ctx) {
 	    super(getProperty(type, ctx, HOST_KEY),
 	          getProperty(type, ctx, USER_KEY),
 	          getProperty(type, ctx, PASSWORD_KEY),
-              SshConfig$.MODULE$.DEFAULT_TIMEOUT());
+              SshConfig$.MODULE$.DEFAULT_TIMEOUT(),
+              ImOption.apply(ctx.getProperty(buildPropertyName(type, ENABLED_KEY))).forall("true"::equals)
```

Comment:
I added an `enabled` bundle to property to make it possible to skip file sftps. This greatly simplifies testing. By default copying is enabled so it should not impact production.